docker/Dockerfile

```diff
@@ -9,7 +9,6 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o /out/usr/bin/protoc-gen-gorm main.go
 FROM infoblox/atlas-gentool:latest AS runner
 
 COPY --from=builder /out/usr/bin/protoc-gen-gorm /usr/bin/protoc-gen-gorm
-COPY --from=builder /go/src/github.com/infobloxopen/protoc-gen-gorm/options/*.proto /go/src/github.com/infobloxopen/protoc-gen-gorm/options
-COPY --from=builder /go/src/github.com/infobloxopen/protoc-gen-gorm/types/*.proto /go/src/github.com/infobloxopen/protoc-gen-gorm/types
```

Comment:
1) why types are not copied anymore?
2) are `.../gen-gorm/options` and `.../gen-gorm/options/` different somehow?

Reply:
@Evgeniy-L Sorry, this was a useless change